apps/condo/lang/ru/messages/PASS_TICKET_COMMENT_CREATED/default.njk

```diff
@@ -0,0 +1 @@
+{{ message.meta.data.senderName }} (житель): {{ message.meta.data.comment }}
```

Comment:
Is it ok that the title, message, and emoji completely match TICKET_COMMENT_CREATED? How will the user distinguish between them?